aseprite.sh

```diff
@@ -123,7 +123,7 @@ export PATH="${PWD}/depot_tools:${PATH}"
 cd skia
 
 # get proper skia's branch to compile
-SKIA_BRANCH=$(curl "https://raw.githubusercontent.com/aseprite/aseprite/master/INSTALL.md" | grep "aseprite-m[0-9][0-9]" | sed -n '1p' | perl -n -e '/(aseprite-m\d\d)/ && print $1')
+SKIA_BRANCH=$(curl "https://raw.githubusercontent.com/aseprite/aseprite/main/INSTALL.md" | grep "aseprite-m[0-9][0-9]" | sed -n '1p' | perl -n -e '/(aseprite-m\d\d)/ && print $1')
```

Comment:
This is arguably still wrong, since it's not using the tag. I think it's probably okay for now.

Reply:
Thanks. This one is good. aseprite really changes the structure of branches included separation of release and beta version.